libraries/AP_Networking/AP_Networking.h

```diff
@@ -157,14 +157,23 @@ class AP_Networking
     enum class OPTION {
         PPP_ETHERNET_GATEWAY=(1U<<0),
 #if AP_NETWORKING_CAN_MCAST_ENABLED
-        CAN1_MCAST_GATEWAY=(1U<<1),
-        CAN2_MCAST_GATEWAY=(1U<<2),
+        CAN1_MCAST_ENDPOINT=(1U<<1),
+        CAN2_MCAST_ENDPOINT=(1U<<2),
+        CAN1_MCAST_BRIDGED=(1U<<3),
+        CAN2_MCAST_BRIDGED=(1U<<4),
 #endif
     };
     bool option_is_set(OPTION option) const {
         return (param.options.get() & int32_t(option)) != 0;
     }
 
+#if AP_NETWORKING_CAN_MCAST_ENABLED
```

Comment:
This would make it onto the bootloader?  Maybe elided?

Separate define as suggested would make this clearly not a problem.

Reply:
done using AP_NETWORKING_CAN_MCAST_BRIDGING_ENABLED